compiler/pom.xml

```diff
@@ -198,6 +202,23 @@
                   <pattern>com.google.auto.common</pattern>
                   <shadedPattern>dagger.shaded.auto.common</shadedPattern>
                 </relocation>
+                <relocation>
+                  <pattern>com.squareup.javapoet</pattern>
+                  <shadedPattern>dagger.shaded.javapoet</shadedPattern>
+                </relocation>
+                <relocation>
+                  <pattern>com.google.googlejavaformat</pattern>
+                  <shadedPattern>dagger.shaded.googlejavaformat</shadedPattern>
+                </relocation>
+                <!-- Brought in by google-java-format 0.1-alpha -->
```

Comment:
Is there a way to allow downstream users to shade what they want if they need to, but if they don't to just use what's provided?